cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/AddCommand.java

```diff
@@ -171,7 +171,7 @@ public void execute() {
         // Check if the template exists
         if (!getTemplates().contains(template)) {
             CommandUtil.printError(errStream,
-                    "Template not found, use `ballerina add --list` to view available templates.",
+                    "Template is not found. Use `ballerina add --list` to view the available templates.",
```

Comment:
IMO "Template not found." is OK and may feel natural to developers. If it doesn't sound right, shall we say "Cannot find the template."

IMO, we should also say what template was not found
 ```suggestion
                    "Cannot find the template '" + template + "'. Use `ballerina add --list` to view the available templates.",
```

Reply:
Changed to "Cannot find this template."